abis_mapping/templates/dwc_mvp/metadata.json

```diff
@@ -3,5 +3,6 @@
     "description": "A template to translate some Darwin Core fields",
     "version": "0.0.1",
     "template_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/dwc_mvp/dwc_mvp.csv",
-    "schema_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/dwc_mvp/schema.json"
+    "schema_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/dwc_mvp/schema.json",
+    "instructions_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/dwc_mvp/README.md"
```

Comment:
Already merged that in https://github.com/gaiaresources/abis-mapping/pull/17.  
Not sure why it's not showing in the diff